python/lsst/dax/imgserv/imageREST_v1.py

```diff
@@ -0,0 +1,226 @@
+#!/usr/bin/env python
```

Comment:
Remove this (file is not executable). Generally, this file has a few PEP8 issues.

Reply:
Removed.  Ran PEP8 on the file.